app/models/refund.rb

```diff
@@ -43,4 +43,40 @@ def eat_issue_fee_present?
   def eat_hearing_fee_present?
     eat_hearing_fee.present? && eat_hearing_fee.positive?
   end
+
+  class << self
+    def to_csv(ids)
```

Comment:
Im not sure I follow this.  Where do ids come from ?  If there is an id for every record in the database, what happens when there are 50,000 of them ? 
The csv is being built in memory rather than streaming to a file directly - again, if we are dealing with 50,000 records, I would be a bit worried about memory usage.  I guess we would need to estimate the size of an average row ?

Reply:
@garytaylor the IDs are from the view. You can't select 50k records. But I guess that you can call the method from another one. But I'm using ```find_each``` method which is iterating in batches. So I would not worry about the memory usage because that's what find_each is for.

Reply:
Does this mean the user can only export a page at a time ?  If so, is that acceptable ?

Reply:
@garytaylor yes that's what it means. There is no AC that would say that it has to be exported all at once by the user. My thinking was to show it to Cathie so I can get some feedback.

Reply:
Ok. let's use service.

Reply:
ok - I would have expected it to export all records, but if that wasnt specified then maybe not - like you say, check with Cathie